qcfractal/storage_sockets/sqlalchemy_socket.py

```diff
@@ -1174,22 +1192,24 @@ def update_results(self, record_list: List[ResultRecord]):
 
         # try:
         updated_count = 0
+        result_list = []
+
         for result in record_list:
 
             if result.id is None:
                 self.logger.error("Attempted update without ID, skipping")
                 continue
-            with self.session_scope() as session:
-
-                result_db = session.query(ResultORM).filter_by(id=result.id).first()
 
-                data = result.dict(exclude={"id"})
-
-                for attr, val in data.items():
-                    setattr(result_db, attr, val)
+                # result_db = session.query(ResultORM).filter_by(id=result.id).first()
+            # data = result.dict()
+            result_list.append(ResultORM(**result.dict()))
+            updated_count += 1
 
-                session.commit()
-                updated_count += 1
+                # for attr, val in data.items():
+                #     setattr(result_db, attr, val)
+        with self.session_scope() as session:
+                session.bulk_save_objects(result_list)
+                
```

Comment:
This appears to have several previous changes in the PR as well. Can you make a new branch and cherry pick your changes over?